Source/Dafny/DafnyPipeline.csproj

```diff
@@ -34,4 +34,19 @@
   <ItemGroup>
     <Content Include="DafnyPrelude.bpl" CopyToOutputDirectory="PreserveNewest" />
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
I don't think DafnyPipeline should reference DafnyRuntime in any way. Can't we have these references in DafnyDriver?

Reply:
I changed my mind on this yet again. The issue I came across is that the `DafnyPipeline` project, which we're making available as a library, includes the compilers. If you invoke those compilers and their assembly (or, as an option, possibly some other one that's available at runtime) doesn't contain the runtime files, they'll crash. So the runtime files can't reasonably go in `DafnyDriver`. They ultimately should go in language-specific compiler packages, as I think we all agree. But I still feel like it'd be messy to put them in the `DafnyRuntime` assembly (even though the source code is in that package). So that leaves `DafnyPipeline`, for now, until we split off the individual compilers.